CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Fix `ChannelDoesNotExist` error is logged by `UserWatchingEventMiddleware` when channels are fetched for the first time [#893](https://github.com/GetStream/stream-chat-swift/issues/893)
 
+⚠️ Breaking Changes ⚠️
+
+- Some model properties became thread-unsafe to improve performance [#906](https://github.com/GetStream/stream-chat-swift/issues/906)
+    - `_ChatChannel.latestMessages`
+    - `_ChatMessage.author`
+    - `_ChatMessage.mentionedUsers`
+    - `_ChatMessage.attachments`
+    - `_ChatMessage.latestReplies`
+    - `_ChatMessage.latestReactions`
+    - `_ChatMessage.currentUserReactions`
+
+
+
```

Comment:
This just a temporary downside, no need to mention it in the changelog. However, you should mention there that the performance is improved due to some properties being loaded lazily :)